src/current/cockroachcloud/upgrade-to-v24.2.md

```diff
@@ -0,0 +1,175 @@
+---
```

Comment:
to confirm:
this is what is in the other PR (https://github.com/cockroachdb/docs/pull/18798/) already right?
assume you added this to get the build green (it didn't work since build failed but I assume that's why :-} )

Reply:
This is just the search-and-replace copy of the file. The tooling and innovation release content is happening separately.